packages/machine/test/integration/test-commitment-store.ts

```diff
@@ -193,13 +195,7 @@ export class TestCommitmentStore {
       );
     }
 
-    const sigs = [signature, counterpartySignature].map(sig => {
-      if (!(sig instanceof cf.utils.Signature)) {
-        const { v, r, s } = sig as any;
-        return new cf.utils.Signature(v, r, s);
-      }
-      return sig;
-    });
+    const sigs = [signature, counterpartySignature];
```

Comment:
Probably unnecessary to declare a variable here and can just pass it into the function below inline

Reply:
ala https://techbeacon.com/why-unnecessary-variables-are-bad-your-code